src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -76,23 +87,51 @@ public void Close()
         }
 
         /// <inheritdoc/>
+        public bool CheckVersionWithTestHost()
```

Comment:
This reminds me - These RFCs([1](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0003-Test-Execution-Protocol.md) ,[2](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0002-Test-Discovery-Protocol.md)) are now out of date. Should those change?